packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -225,9 +209,7 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
         break;
       case 'touchstart':
       case 'touchmove':
-        currentPosition = !vertical
-          ? (event as TouchEvent).touches[0].clientX
-          : (event as TouchEvent).touches[0].clientY;
+        currentPosition = !vertical ? (event as TouchEvent).touches[0].clientX : (event as TouchEvent).touches[0].clientY;
```

Comment:
It looks like a lot of these changes are just reformatting; do we want them or your settings differ from what is checked-in and should be reverted?

Reply:
They're actually a result of someone changing the prettier config line-width to 140 but not running prettier on the file :( 

It results in a lot of unfortunate superfluous whitespace changes. I believe @khmakoto will be looking to fix this.

cc @dzearing